LEGO1/tgl/d3drm/group.cpp

```diff
@@ -0,0 +1,129 @@
+#include "impl.h"
+
+using namespace TglImpl;
+
+// Inlined only
+GroupImpl::~GroupImpl()
+{
+	if (m_data) {
+		free(m_data);
```

Comment:
This should be `delete`

Reply:
Fixed, should actually have been `Release()`